treeBuild.sh

```diff
@@ -0,0 +1 @@
+cat treeBuild/*.js >> build/build_in_tree.js
```

Comment:
Let's call this built_in_trees.js to be more clear that the file contains many trees.